api/src/pcapi/routes/backoffice/gdpr_user_extract/blueprint.py

```diff
@@ -0,0 +1,52 @@
+from flask import render_template
+import sqlalchemy as sa
+
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.users import models as users_models
+from pcapi.models.feature import FeatureToggle
+from pcapi.routes.backoffice import utils
+
+
+gdpr_extract_blueprint = utils.child_backoffice_blueprint(
+    "gdpr_extract",
+    __name__,
+    url_prefix="/gdpr-extract",
+    permission=perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT,
+)
+
+
+def _get_gdpr_data() -> list[users_models.GdprUserDataExtract]:
+
+    base_query = (
+        users_models.GdprUserDataExtract.query.options(
+            sa.orm.joinedload(users_models.GdprUserDataExtract.user).load_only(
+                users_models.User.firstName,
+                users_models.User.lastName,
+            ),
+            sa.orm.joinedload(users_models.GdprUserDataExtract.authorUser).load_only(
+                users_models.User.firstName,
+                users_models.User.lastName,
+            ),
+        )
+        .order_by(users_models.GdprUserDataExtract.id.desc())
+        .filter(users_models.GdprUserDataExtract.expirationDate >= sa.func.now())
+    )
+
+    query = base_query.all()
+    return query
```

Comment:
Je ne comprends pas bien : `query` n'est pas une requête mais contient les résultats.
Tu écris `base_query` mais ce n'est pas juste la base, c'est tout la requête.
Renommer `base_query` en `query` serait plus clair, et ensuite : 
```suggestion
    return query.all()
```